### PR TITLE
[New] Created Lightbox and Gallery Component

### LINKED ISSUE #24: Alex - working on lightbox

### PR DESCRIPTION
**Pull request by YaBoiBreezy**
⭐ **New Features:**

- description of feature 1
- description of feature 2

ℹ️ **Related Issues:**

resolves #24 

📷 **Screenshots:** _(if applicable)_

### CODE REVIEW

**Comment by YaBoiBreezy** [comment]
Done, remaining errors are surface-level / inconsequential, tests are only at like 50% maybe. Will keep working on it. The actual component and stories are done though, could honestly merge anytime IMO

**Comment by YaBoiBreezy** [comment]
Done the component, also has upgrades for link and image. IDK why chromatic test fails, idk how to fix

File: src/components/index.ts [hunk 1]
```diff
@@ -18,5 +18,4 @@ export { default as Jumbotron } from './Jumbotron';
 export { default as Table } from './Table';
 export { default as Modal } from './Modal';
 export { default as InputGroup } from './InputGroup';
-export { default as Grid } from './Grid';
-export { default as List } from './List';
```
Comment (Rizvida1) [inline src/components/index.ts:22]: Should these be removed?

File: tsconfig.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
     "skipLibCheck": true,
     "baseUrl": "./",
     "paths": {
-      "@components/*": ["src/components/*"]
+      "@components/*": ["src/components/*"],
+      "@storybook/*": [".storybook/*"]
```
Comment (Rizvida1) [inline tsconfig.json:12]: Is there a reason for this?

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

File: src/components/index.ts (hunk 1 above)
Comment (YaBoiBreezy) [reply src/components/index.ts:22]: those are 2 components that got merged with master this morning, it made conflicts with a bunch of components. I will try allowing them

File: src/components/Lightbox/LightboxGallery.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+          </button>
+          <button
+            title="Next (right arrow key)"
+            type="button"
+            className="mfp-arrow mfp-arrow-right mfp-prevent-close"
+            onClick={() => {
+              setIndex((index + 1) % numImages);
+            }}
+          >
+            <span className="wb-inv"> Next (right arrow key)</span>
```
Comment (Rizvida1) [inline src/components/Lightbox/LightboxGallery.tsx:144]: Will this text translate if french is needed?

**Review by Rizvida1 — APPROVED** [review]

File: src/components/Lightbox/LightboxGallery.tsx (hunk 3 above)
Comment (Rizvida1) [inline src/components/Lightbox/LightboxGallery.tsx:144]: Will this text translate if French is needed?

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

File: src/components/Lightbox/LightboxGallery.tsx (hunk 3 above)
Comment (YaBoiBreezy) [reply src/components/Lightbox/LightboxGallery.tsx:144]: No, adding in French version today

**Review by Rizvida1 — APPROVED** [review]

**Pull request merged**